arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -190,6 +190,8 @@ RocksDBEngine::RocksDBEngine(application_features::ApplicationServer& server)
 #endif
       _useThrottle(true),
 #ifdef USE_ENTERPRISE
+      // TODO: shall we turn this on by default for all enterprise installations,
```

Comment:
Understood, but as this is still a configurable option which users can turn off, I repeat my question:
do we want the default to be "on" or "off". If we say on, fine for me, but it will create (useless) background work when RocksDB finishes writing SST files. This work is completely unnecessary if the hotbackup is never used.